src/Storefront/Resources/views/storefront/section/cms-section-sidebar.html.twig

```diff
@@ -3,30 +3,35 @@
     {# ludtwig-ignore twig-hash-key-no-quotes #}
     {% set layout = section.sizingMode ? section.sizingMode|replace({'_': '-'}) : 'container' %}
 
-    <div class="cms-section-sidebar {{ sectionMobileBehaviorClass }} {{ layout }} row">
+    <div class="cms-section-sidebar {{ sectionMobileBehaviorClass }} {{ layout }}">
 
-        {% set sidebarBlocks = section.blocks.filterBySectionPosition('sidebar') %}
-        {% set mainContentBlocks = section.blocks.filterBySectionPosition('main') %}
+        {% block page_content_section_sidebar_row %}
+            <div class="row">
 
-        {% block section_sidebar_content %}
-            <div class="cms-section-sidebar-sidebar-content col-lg-4 col-xl-3">
+                {% set sidebarBlocks = section.blocks.filterBySectionPosition('sidebar') %}
+                {% set mainContentBlocks = section.blocks.filterBySectionPosition('main') %}
 
-                {% for block in sidebarBlocks %}
-                    {% block section_sidebar_content_block %}
-                        {% sw_include '@Storefront/storefront/section/cms-section-block-container.html.twig' %}
-                    {% endblock %}
-                {% endfor %}
-            </div>
-        {% endblock %}
+                {% block section_sidebar_content %}
+                    <div class="cms-section-sidebar-sidebar-content col-lg-4 col-xl-3">
+
+                        {% for block in sidebarBlocks %}
```

Comment:
```suggestion
                        {% for currentBlock in sidebarBlocks %}
```
I'd prefer not using a keyword for the current element. Would be nice if you change that as well, when you're already at it 😄

Reply:
I agree, using the keyword is a bit confusing with the syntax highlighting. Interesting, that this does not trigger any warning or error.

But I think it's unrelated to your change and we could also do it separately if boy-scouting this has some side-effects (is the block variable used in the include templates?)

Reply:
In general I agree, but I would not change this now, as it would break the scope of this change, and in addition, it would be a break that was not deprecated beforehand. The block variable inherits down to the other templates and had to be changed in many places.